docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md

```diff
@@ -1,48 +1,158 @@
-import UseTheBuiltInHelpToReviewTheCliSSuiteNewOptionalFlags from './components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_use_the_builtin_help_to_review_the_clis_suite_new_optional_flags.mdx'
+import Prerequisites from '/docs/components/_prerequisites.jsx'
+import PrerequisiteQuickstartGuideComplete from '/docs/components/prerequisites/_quickstart_completed.mdx'
+import IfYouStillNeedToSetupGX from '/docs/components/prerequisites/_if_you_still_need_to_setup_gx.md'
+import DataContextInitializeQuickOrFilesystem from '/docs/components/setup/link_lists/_data_context_initialize_quick_or_filesystem.mdx'
+import ConnectingToDataFluently from '/docs/components/connect_to_data/link_lists/_connecting_to_data_fluently.md'
 
-<Preface />
+In order to Validate data, we must first define a set of Expectations for that data to be validated against.  In this guide, we will walk you through processes of interactively creating and editing Expectations while simultaneously validating them against a Batch of data. By validating our Expectations as we define them we can quickly see if the Expectations we are setting are suitable for our data, and adjust them if necessary.
+
+:::tip Does this process edit my data?
+Not at all.  We call this process the "Interactive method" of creating and editing Expectations because validating the Expectations as we work requires interacting with a Batch of data.  This data is introspected to see if it fits the defined Expectation, but it is not edited: only examined.
+:::
+
+## Prerequisites
+
+<Prerequisites>
+
+- Great Expectations installed in a Python environment
+- Initialized a Filesystem Data Context to save your Expectations in
+- Created a Datasource from which to request a Batch of data for introspection
+- A passion for data quality
+
+</Prerequisites> 
+
+<details>
+<summary>
+
+### If you still need to set up Great Expectations...
+
+</summary>
+
+<IfYouStillNeedToSetupGX />
+
+</details>
+
+<details>
+<summary>
+
+### If you still need to initialize your Data Context...
+
+</summary>
+
+Please see the appropriate guide from the following:
+
+<DataContextInitializeQuickOrFilesystem />
+
+</details>
+
+<details>
+<summary>
+
+### If you still need to create a Datasource...
+
+</summary>
+
+Please reference the appropriate guide from the following:
+
+<ConnectingToDataFluently />
+
+</details>
 
 ## Steps
 
-### 1. Use the CLI to begin the interactive process of creating Expectations
-<UseTheCliToBeginTheInteractiveProcessOfCreatingExpectations />
+### 1. Import the Great Expectations module and instantiate a Data Context
+
+For this guide we will be working with Python code in a Jupyter Notebook. Jupyter is included with GX and provides a very convenient interface that lets us easily edit code and immediately see the result of our changes.
```

Comment:
```suggestion
To complete the procedures in this guide, you'll use Python code in a Jupyter Notebook. Jupyter is included with GX and it lets you edit code and immediately see the results of your changes.
```
What are users doing here? Tell them what they'll accomplish - _You'll use Python code in a Jupyter Notebook to..._

![image](https://user-images.githubusercontent.com/104205257/231879477-9680f7b3-d0b8-488b-abf0-38c372296fec.png)